page/partials/quotes.ejs

```diff
@@ -0,0 +1,19 @@
+<div class="section quotes">
+  <div class="row">
+    <div class="col-md-6">
+      <h3><%= title %></h3>
```

Comment:
is h3 without any additional styles look good or should i style it since i do it for all other elements?

Reply:
Yes it looks same as other section titles